src/Illuminate/Container/Container.php

```diff
@@ -581,9 +581,11 @@ public function wrap(Closure $callback, array $parameters = [])
      * Call the given Closure / class@method and inject its dependencies.
      *
      * @param  callable|string  $callback
-     * @param  array  $parameters
+     * @param  mixed[string]  $parameters
```

Comment:
Invalid syntax.

Reply:
will double check it.

Reply:
Does this array<string, mixed> do the job?
I saw something like this on the Str::parseCallback docblock

Reply:
Yes.